spec/index.html

```diff
@@ -428,16 +426,14 @@ <h2>SPARQL Protocol Operations</h2>
         <li>The message content included in the HTTP request body.</li>
         <li>The message content included in the HTTP response body.</li>
       </ul>
-      <p>The SPARQL 1.1 Protocol is built on top of HTTP. All HTTP requirements for requests and responses <strong>must</strong> be followed.</p>
+      <p>The SPARQL 1.2 Protocol is built on top of HTTP. All HTTP requirements for requests and responses <strong>must</strong> be followed.</p>
       <section id="query-operation">
         <h3>Query Operation</h3>
         <p>The <code>query</code> operation is used to send a SPARQL query to a service and receive the results of the query. The query operation <strong>MUST</strong> be invoked with either the HTTP
-          GET or HTTP POST method. Client requests for this operation <strong>must</strong> include exactly one SPARQL query string (parameter name: <code>query</code>) and <strong>may</strong> include
-          zero or more default graph URIs (parameter name: <code>default-graph-uri</code>) and named graph URIs (parameter name: <code>named-graph-uri</code>). The response to a query request is either
-          the <a href="http://www.w3.org/TR/rdf-sparql-XMLres/">SPARQL XML Results Format</a>, the <a href="http://www.w3.org/TR/sparql11-results-json/">SPARQL JSON Results Format</a>, the 
-          <a href="http://www.w3.org/TR/sparql11-results-csv-tsv/">SPARQL CSV/TSV Results Format</a>, or an RDF serialization, depending on the 
-          <a href="http://www.w3.org/TR/sparql11-query/#QueryForms">query form</a> [<a href="#sparql">SPARQL</a>] and 
-          <a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html">content negotiation</a> [<a href="#rfc2616">RFC2616</a>].
+        GET or HTTP POST method. Client requests for this operation <strong>must</strong> include exactly one SPARQL query string (parameter name: <code>query</code>) and <strong>may</strong> include
+        zero or more default graph URIs (parameter name: <code>default-graph-uri</code>) and named graph URIs (parameter name: <code>named-graph-uri</code>). The response to a query request is either
+        the [[[SPARQL12-RESULTS-XML]]], the [[[SPARQL12-RESULTS-JSON]]], the [[[SPARQL12-RESULTS-CSV-TSV]]], or an RDF serialization, depending on the <a data-cite="SPARQL12-QUERY#QueryForms">query
+        form</a> and <a data-cite="rfc9110#name-content-negotiation">content negotiation</a>.
```

Comment:
Each RFC 9110 section have a number anchor and name anchor. We can use either, and try to be consistent (it's not a disaster if we're not).

Their TOC uses both - "section" for numbering and "name" for the text in the TOC list item

`#section...` is to the left of the name anchor.

Here it would be:

```suggestion
        form</a> and <a data-cite="rfc9110#section-12">content negotiation</a>.
```